tests/tests/test_monitor_client.py

```diff
@@ -249,6 +251,23 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
         )
         time.sleep(wait_for_alert_interval_s)
 
+        r = inventory.get_device(devid)
+        assert r.status_code == 200
+        inventory_data = r.json()
+        alert_count = alerts = None
+        for inventory_item in inventory_data["attributes"]:
+            if (
+                inventory_item["scope"] == "monitor"
+                and inventory_item["name"] == "alert_count"
+            ):
+                alert_count = inventory_item["value"]
+            elif (
+                inventory_item["scope"] == "monitor"
+                and inventory_item["name"] == "alerts"
+            ):
+                alerts = inventory_item["value"]
+        assert (1, True) == (alert_count, alerts)
```

Comment:
this is repeated, so maybe a helper method? something accepting devid, alerts_count int and alerts boolean that performs the query and checks?

Reply:
Ok, let's do it